packages/babel-plugin-transform-classes/src/vanilla.js

```diff
@@ -200,22 +144,20 @@ export default class ClassTransformer {
   }
 
   /**
-   * [Please add a description.]
-   * https://www.youtube.com/watch?v=fWNaR-rxAic
+   * Creates a class constructor if there is none
    */
-
-  constructorMeMaybe() {
+  function createConstructor() {
```

Comment:
nitpick - from what I observed `maybe*` is a convention used in babel's codebase to indicate that a function attempts to do something, but may bail out - imho it might be good to preserve it here as this function won't always "create constructor" 😉

Reply:
Renamed it to `maybeCreateConstructor`. See last 2 commits.